package.json

```diff
@@ -3,28 +3,28 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
-    "@adobe/aio-lib-core-logging": "^2.0.0",
     "@adobe/aio-sdk": "^3.0.0",
-    "@adobe/exc-app": "^0.2.17",
-    "cloudevents": "^4.0.2",
     "node-fetch": "^2.6.0",
+    "cloudevents": "^4.0.2",
+    "uuid": "^8.0.0",
+    "core-js": "^3.6.4",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
where are the react libs used?

Reply:
they got added because of the web-src default aio project builds. I will remove them